### ISSUE TITLE
Epic: Instrument "Onboarding" Growth Area Telemetry Data in IDE

### ISSUE THREAD

**Issue by loujaybee** [gitpod-io/gitpod#7182]
| Context             | We are aiming to improve the onboarding success rate into Gitpod (defined as the steps between sign-up and commit in a workspace) within the IDE team. This epic is to lay the groundwork / enabling work, such as adding data and analytics tracking to hosted VSCode. |
|---------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Owner               | Product - @loujaybee \ Tech - @akosyakov                                                                                                                                                                                                                                |
| Value               | Better inform product decisions on the Gitpod onboarding phase.                                                                                                                                                                                                         |
| Acceptance Criteria | - Define an "endpoint" metric for the onboarding flow which defines when "onboarding" is complete. Could be a file edit in VS Code: #6896 or a git command #6981 such as commit.  |
| Growth Area         | `Onboarding`                                                                                                                                                                                                                                                            |
| Persona(s)          | N/A                                                                                                                                                                                                                                                                     |
| Hypothesis          | N/A                                                                                                                                                                                                                                                                     |
| Measurement         | N/A                                                                                                                                                                                                                                                                     |
| In scope            | - Instrumentation of analytics                                                                                                                                                                                                                                          |
| Out of scope        | - Any functional changes to Gitpod<br>- Instrumenting / measuring desktop IDE's                                                                                                                                                                                         |
| Complexities        | N/A                                                                                                                                                                                                                                                                     |

**Comment by loujaybee** [comment]
We need to integrate this endpoint now into Mixpanel, but I think we're now waiting on @jakobhero to resolve some issues with Mixpanel vs Mode. 

CC: @phimae

**Comment by loujaybee** [comment]
Whilst we are still waiting on this to be integrated into PLG areas dashboard [[1](https://www.notion.so/gitpod/PLG-Areas-Dashboard-6ff7bba7eec6413587ac5be7c3fde33a)] I'm going to adjust the scope to be only the instrumentation, and not the visualisation so that we can close this issue now, and raise follow up issues for integration with visualisations / dashboards.
